apps/frontend/src/components/form/types.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Common props for all form field components.
+ */
+export interface GenericFormFieldProps {
```

Comment:
Ohh, I think these were there before for validating the inputs, but we decided that we didn't need to validate for now. I'll delete this!

Reply:
I used the top interface (GenericFormFieldProps) in InputField.tsx to place some constraints on the user input like I made isRequired true for every input box on the form so the user has to fill out the form completely to submit it. I'm not sure if that's necessary though, let me know if it would be cleaner to delete it

Reply:
i think removing it is the right call -- it seems like kind of unnecessary bulk for the simplicity of the form we're working on